New feature: CLM filter templates (#942)
uyuni-project/uyuni-docs: modules/administration/pages/content-lifecycle.adoc

[#filter-templates]
== Filter Templates

To help with creating filters for some common scenarios, {productname} offers filter templates.

> When applied, these templates help create a set of pre-defined filters, tailored for a specific use case.

Reviewer: ```suggestion
When applied, these templates help creating a set of filters in advance, tailored for a specific use case.
```

This section describes available templates and their usages.

=== Live patching based on a SUSE product

In a project that contains live patching, regular kernel packages must be excluded so that only live patch packages are offered as updates to clients. On the other hand, already installed regular kernel packages must still be included to keep system integrity.

When applied, this template creates two filters required to achieve this behavior:

> * Deny patches that contain `kernel-default` package greater than a specific kernel version

Reviewer: I think asciidoc wants two tics...
```suggestion
* Deny patches that contain ``kernel-default`` package greater than a specific kernel version
```

> * Deny patches that contain `reboot_suggested` keyword

Reviewer: ```suggestion
* Deny patches that contain ``reboot_suggested`` keyword
```

[IMPORTANT]
====

> The second "catch-all" filter is a safeguard to make sure the assigned clients are never required a restart in the future. You can choose not to attach it to the project if this behavior is not desired as it might block other important updates.

Reviewer: We have this strange policy (I think it help adoc editors): New sentence = new line
```suggestion
The second "catch-all" filter is a safeguard to make sure the assigned clients are never requiring a restart in the future.
You can choose not to attach it to the project if this behavior is not desired because it might block other important updates.
```

====

For more information on how to set up a live patching project, see xref:administration:content-lifecycle-examples.adoc#exclude-higher-kernel-version[].

.Procedure: Applying the template
. In the {productname} {webui}, navigate to menu:Content Lifecycle[Filters], and click btn:[Create Filter].
. In the dialog, click btn:[Use a template]. The inputs will change accordingly.
. In the [guimenu]``Prefix`` field, type a name prefix. This value will be prepended to the name of every individual filter created by the template. If the template is being applied in the context of a project, this field will be prefilled with the project label.
. In the [guimenu]``Template`` field, select [guimenu]``Live patching based on a SUSE product``.
. In the [guimenu]``Product`` field, select the product you wish to set up live patching for.
. In the [guimenu]``Kernel`` field, select a kernel version from the list of versions available in the selected product. The filter to deny the later regular kernel patches will be based on this version.
. Click btn:[Save] to create the filters.
. Navigate to menu:Content Lifecycle[Projects] and select your project.
. Click btn:[Attach/Detach Filters].

> . Select the 2 filters that have the specified prefix, and click btn:[Save].

Reviewer: ```suggestion
. Select the two filters that have the specified prefix, and click btn:[Save].
```
